lineax/_misc.py

```diff
@@ -45,9 +45,15 @@ def __call__(self, *args, **kwargs):
         return self.fn(*args, **kwargs), None
 
 
-def jacobian(fn, in_size, out_size, holomorphic=False, has_aux=False):
+def jacobian(fn, in_size, out_size, holomorphic=False, has_aux=False, jac=None):
     # Heuristic for which is better in each case
     # These could probably be tuned a lot more.
+    if jac is not None:
+        if jac == "bwd":
+            return jax.jacrev(fn, holomorphic=holomorphic, has_aux=has_aux)
+        if jac == "fwd":
+            return jax.jacfwd(fn, holomorphic=holomorphic, has_aux=has_aux)
+
```

Comment:
I think this can be made a little more robust to passing invalid arguments:
```python
if jac is None:
    jac_fwd = (in_size < 100) or (in_size <= 1.5 * out_size)
elif jac == "fwd":
    jac_fwd = True
elif jac == "bwd":
    jac_fwd = False
else:
    raise ValueError("`jac` should either be None, 'fwd', or 'bwd'.")
if jac_fwd:
    return jax.jacfwd(...)
else:
    return jax.jacrev(...)
```